container/memory_store.go

```diff
@@ -25,8 +28,13 @@ func (c *memoryStore) Add(id string, cont *Container) {
 
 // Get returns a container from the store by id.
 func (c *memoryStore) Get(id string) *Container {
+	var res *Container
 	c.RLock()
-	res := c.s[id]
+	if runtime.GOOS == "solaris" {
+		res = c.s[id[:12]]
```

Comment:
I'll remove this. This is a remnant from a time when the runtime on Solaris couldn't handle the full 256 SHA in the container name. That has been fixed. Its been reverted in most places. I missed these two. Thank you for pointing it out!